tests/integrations/test_dsm.py

```diff
@@ -66,48 +81,43 @@ class Test_DsmRabbitmq:
     """ Verify DSM stats points for RabbitMQ """
 
     def setup_dsm_rabbitmq(self):
-        self.r = weblog.get("/dsm?integration=rabbitmq")
+        self.r = weblog.get(
+            f"/dsm?integration=rabbitmq&queue={DSM_QUEUE}&exchange={DSM_EXCHANGE}&routing_key={DSM_ROUTING_KEY}"
+        )
 
-    @bug(library="dotnet", reason="bug in dotnet behavior")
+    @bug(
+        library="java",
+        reason="Java calculates 16129003365833597547 as producer hash by not using 'routing_key:true' in edge tags.",
+    )
+    @bug(
+        library="dotnet",
+        reason="Dotnet calculates 3168906112866048140 as producer hash by using 'routing_key:True' in edge tags, with 'True' capitalized, resulting in different hash.",
+    )
```

Comment:
Should we register this bug in Jira? or is it more related with the test?

Reply:
I've asked @vandonr to take a look since he's an expert in those tracers. If they are found to be actual bugs, we'll make sure to add some Jira tickets